redhat-developer/web-terminal-tooling
# Install multiple cluster tools along with bash-completion for them

### What does this PR do?

This PR installs multiple cluster tools along with bash-completion for them:

There is a test built image: sleshchenko/web-terminal-tooling:ubi8 538MB. That contains the following tools (note that size are approximated):
- registry.access.redhat.com/ubi8-minimal +146Mb
- bash-completion ncurses pkgconf-pkg-config git procps +100Mb
- jq +0 included in ubi8 image
- oc +75Mib
- kubectl +75Mib
- odo +66.6Mib
- helm +36.7Mib
- Tekton CLI +37.3Mib
- kubectx & kubens +700Kib

Dropped:
- yq -46Mb (along with pip that is needed to install it)
- istioctl -105.4Mib
- crwctl -95.3Mib

To test built image 
````bash
podman pull --tls-verify=false registry-proxy.engineering.redhat.com/rh-osbs/web-terminal-web-terminal-tooling:web-terminal-1.0-rhel-8-containers-candidate-76346-20200622102429
podman run -ti registry-proxy.engineering.redhat.com/rh-osbs/web-terminal-web-terminal-tooling:web-terminal-1.0-rhel-8-containers-candidate-76346-20200622102429 bash
```
To build image by yourselves - just ./build.sh from repo.

### Which issue does it reference/solve?
It's done in the scope of https://github.com/eclipse/che/issues/16938

### What is not addressed?

mc
